3478/2/utils/webhooks_controller.rb

```diff
@@ -0,0 +1,15 @@
+require_relative 'command/start_command.rb'
+require_relative 'command/unlink_command.rb'
+require_relative 'command/checkin_command.rb'
+require_relative 'command/checkout_command.rb'
+require_relative 'file_manager.rb'
+
+class WebhooksController < Telegram::Bot::UpdatesController
+  Telegram::Bot::UpdatesController.session_store = :redis_store, { expires_in: 2_250_000 }
+  include Telegram::Bot::UpdatesController::MessageContext
+
+  include StartCommand
```

Comment:
And why I need all includes here?

Reply:
this is required by the gem, so i can't change it

Reply:
It's about your understanding of what are you doing.  You can collect all your modules in 1 and include it everywhere(it's bad, but in this case it's ok) `CommandHelper` `UtilHelper` etc
You can read about helpers ;)

Reply:
Makes sense

Reply:
so do this